### ISSUE TITLE
关于match_loss的疑问

### ISSUE THREAD

**Issue by clarkhedi** [CFM-MSG/Code-AUL#3]
您好，非常感谢您的开源工作！
在您的代码中，损失计算中多加了一个关于图像特征和文本特征之间的 match_loss，具体计算如下图：
![image](https://github.com/user-attachments/assets/f2fa3d3e-3f39-4d3a-a118-5f2cc3a01217)

对于其中的 scores 的维度是 2 × batch_size，而 label 的值为 [1, 0]，这两者计算 F.cross_entropy(scores, label)，这里相当于2个样本，第1个样本只计算scores中index为1的值？第2个样本只计算scores中index为0的值？

感谢回复！
